### ISSUE TITLE
Unknown Owner: UUID.offline: true

### ISSUE THREAD

**Issue by TagCraftMC** [IntellectualSites/PlotSquared#624]
Greetings,

Recently I changed the setting UUID.offline: false to UUID.offline: true, afterwards I started to notice (as well as get reports) that people's plots were being transferred to an "unknown" entity.
I checked the DB and got the UUID for one of the spawn plots (under my UUID) and it's still 16543f88-e515-3582-a206-1d1cdb04823c (my offline UUID).
There seems to be no errors in console.

Let me know if there is any more info I can provide.
Thanks.

Image of the plot: https://gyazo.com/7837dfe091349e4adb72a1eb6bbbd8a0
DB Dump: http://www.filedropper.com/creativeplotsquared1 (SQL)
Version: Commit 9682e2914ca41c8c357476884dcc6b460f75cd44

**Comment by boy0001** [comment]
Can you check what UUID it is caching for you?
`/plot debugexec seen <player>`

**Comment by L4BORG** [comment]
I'm having the same problem, except I've been using clean install of PlotSquared and had "UUID.offline: true" right from the start....

and if I try /plot debugexec seen <player> I get:

```
[P2] player hasn't connected before: username
```

I've also got no errors and from what I've seen it usually happens after restart... but I can't really point out why this is happening only for some players.

Now I'm wondering if usernames are stored somewhere in case we have "UUID.offline: true"? And if so where would that be? Since maybe didn't get "deowned" and I haven't join with that acc. for days...

Also 
```
[P2] player hasn't connected before: username
```
would change and fix plot owner after player joined the server (first time since last restart) but only for some players. I'll report back when I find next bugged player :)

**Comment by boy0001** [comment]
@L4BORG 
You weren't meant to literally use `/plot debugexec seen username`, you would instead use `/plot debugexec seen Notch` (or some other username).

Usernames are stored in the playerdata files, however in offline-mode, the uuid can be generated from the username without these files.

A common issue with offline-mode is case sensitivity, so you could attempt to convert to lowercase uuids: `/plot uuidconvert lowercase`

@TagCraftMC 
Haven't heard a response, is it still an issue?

**Comment by MattBDev** [comment]
merged with #781
